XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -2318,6 +2318,37 @@
           ]
         }
       ]
+    },
+    {
+      "UniqueId": "TipDocs",
+      "Title": "Tips and Guides",
+      "Subtitle": "",
+      "ImagePath": "",
+      "Description": "",
+      "Items": [
+        {
+          "UniqueId": "CornerRadius",
+          "Title": "CornerRadius",
+          "Subtitle": "Round Corners",
+          "ImagePath": "ms-appx:///Assets/DefaultIcon.png",
+          "Description": "You may set CornerRadius property starting in WinUI 2.2. It sets round corners to your control",
```

Comment:
CornerRadii is a feature introduced with Windows 10 1809, WinUI 2.2 is just the first version of WinUI that uses them.